src/coreComponents/dataRepository/unitTests/testXmlWrapper.cpp

```diff
@@ -175,6 +175,175 @@ TEST( testXmlWrapper, TensorIllFormed )
   testTensorIllFormed< real64, 3 >( "{ 1.0, 2.O, 3.0 }" ); // invalid floating point value
 }
 
+
+// Attribute format tests
+class real64AttributeTestFixture : public ::testing::TestWithParam< std::tuple< string, real64, bool > >
+{
+protected:
+  string attributeString;
+  real64 expectedValue;
+  real64 parsedValue;
+  bool failureFlag;
+
+  void parseString()
+  {
+    GEOSX_LOG_RANK_0( "Parsing string: " << attributeString );
+    xmlWrapper::stringToInputVariable< real64 >( parsedValue, attributeString );
+    ASSERT_NEAR( expectedValue, parsedValue, 1e-10 );
```

Comment:
Unless it causes any trouble, I would maybe suggest `EXPECT_DOUBLE_EQ` for this, to avoid having to choose absolute tolerances.

(As a side note, I prefer `EXPECT` macros over `ASSERT` ones, unless I really need to terminate the test early. Not that it matters here.)